cmd/env/cancel.go

```diff
@@ -59,10 +60,12 @@ func cancelEnvironmentByID(id string) error {
 		params["org"] = org
 	}
 
-	body, err := client.Do(http.MethodPost, uri.CreateResourceURI("cancel", "environment", id, "", params), nil)
+	_, err = client.Do(http.MethodPost, uri.CreateResourceURI("cancel", "environment", id, "", params), nil)
```

Comment:
Close the body, even if you're not reading it.

Reply:
In all these cases, the body is closed earlier in the requests package: https://github.com/shipyard/shipyard-cli/blob/6cafb5a7331d4612eb71a5a7ddb9f2ab389fdd83/requests/requests.go#L78.
The body here is already a slice of bytes, and the actual response Body had already been closed.